src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -114,9 +114,9 @@ def assertCbsdRecord(self, registration_request, grant_request, grant_response,
             
             # if callSign exist, it should have the same value as registered
             if 'callSign' in cbsd_record[0]:
-                self.assertEqual(device['callSign'], cbsd_record[0]['callSign'])
-            max_eirp_by_MHz = 37;           
-            # if eirpCapability in the record, it should be the same as device or in accepted limits
+                self.assertEqual(device['callSign'], cbsd_record[0]['callSign'])      
+            # if eirpCapability in the record, it should be the same as device or less than Fcc maxEirp value and the value limits of the spec
```

Comment:
Slightly clearer: `... less than the FCC maxEirp value which was injected at the beginning of the test and less than ...`